clusterDeployer.py

```diff
@@ -113,6 +115,7 @@ def remove_master_dhcp_entries(self, hosts: List[ClusterHost]) -> None:
             h.prepare_remove_master_dhcp_leases(self._local_host.bridge, removed_macs)
         lh = host.LocalHost()
         lh.run("systemctl restart libvirtd")
+        self._local_host.ensure_linked_to_network(self._local_host.bridge)
```

Comment:
Why do we need to add the api port to the virbr after removing dhcp entries?

Reply:
virsh net-destroy net-start has a side effect of removing the api port from virbr0. Now, in this case (removing master vm), it is not necessary to add the api port back as we remove the whole cluster anyway ...